config/internal_router/gen_keys.sh

```diff
@@ -56,17 +56,17 @@ EOF
 )"
 
 mkdir -p clients
-for team_id in $(seq 1 "$1"); do
-    team_ip="${TEAM_IP_PREFIX}${team_id}"
-    echo $team_ip
+for checker_id in $(seq 1 "$1"); do
+    checker_ip="${CHECKER_IP_PREFIX}${checker_id}"
+    echo $checker_ip
     privkey=$(wg genkey)
     echo $privkey
     pubkey=$(echo "$privkey" | wg pubkey)
     echo $pubkey
-    echo $team_id
-    cat > "clients/team${team_id}.conf" <<EOF
+    echo $checker_id
+    cat > "clients/team${checker_id}.conf" <<EOF
```

Comment:
this should be called `checkerX.conf` and updated in `terraform/bambictf.tf` as well